apps/step-function.json.j2

```diff
@@ -4,7 +4,11 @@
     "SET_DEFAULT_RESULTS": {
       "Type": "Pass",
       "Result": {
-        "processing_time_in_seconds": 0
+        "processing_time_in_seconds": 0,
+        "get_files": {
+          "logs": [],
+          "expiration_time": null
```

Comment:
Not sure what a good default `expiration_time` should be. I want to take a closer look at what assumptions `hyp3_sdk` makes about that field.

Reply:
Looks like it will have the same behavior whether the `expiration_time` field does not exist or is `None`. I can write additional tests for `hyp3-sdk` to cover the case where it is `None`.